package.json

```diff
@@ -66,5 +68,8 @@
     "tslib": "^2.3.1",
     "typescript": "~4.4.4",
     "webpack": "^5.69.1"
+  },
+  "overrides": {
+    "react-dom": "^17.0.2"
```

Comment:
@fhlavac do we have an explicit reason why we are adding this override? What package is pulling in the wrong version of react-dom?

Reply:
@florkbr 
```
➤ YN0000: @monorepo/root@workspace:. provides react-dom@npm:17.0.2 [2e9c5] with version 17.0.2, which doesn't satisfy the following requirements:

➤ YN0000: react-virtualized@npm:9.22.3 [2e9c5] → ^15.3.0 || ^16.0.0-alpha ✘
```

Unfortunately, `react-virtualized` library does not support the new version of `react-dom` (in latest stable version which is more than a year old). In the last months they don't publish stable releases for some reason :worried:

Reply:
Hm. Alright. We may need to bring this up with the PF team - and maybe possibly contribute ourselves. CC @psav